app/helpers.php

```diff
@@ -1,5 +1,18 @@
 <?php
 
+use Illuminate\Support\Str;
+
+/**
+ * Returns whether email is a test email.
+ *
+ * @param string $email
+ * @return boolean
+ */
+function is_test_email($email)
+{
+    return Str::contains($email, ['thing.org', '@dosome','rockthevote.com', 'test', '+']);
```

Comment:
I wonder what the thinking was behind these separate `thing.org` and `@dosome` checks… can we explicitly check for full `@dosomething.org` and `@example` emails instead? (I believe that's how we filter out test users in Quasar.)

I'm also a little concerned about the overly generous `test` and `+` checks. For example, thinking how this might trigger unexpectedly on a user like "Mitt Ester" with the email `mittester@gmail.com`, or folks who like to use Google's [special `+` logic](https://gizmodo.com/how-to-use-the-infinite-number-of-email-addresses-gmail-1609458192) to filter out marketing emails.

(I know this is just refactoring the existing logic into a helper, but it caught my eye!)